Containerfile.aws-load-balancer-operator-bundle

```diff
@@ -38,6 +42,29 @@ LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.16.0+git
 LABEL operators.operatorframework.io.metrics.mediatype.v1=metrics+v1
 LABEL operators.operatorframework.io.metrics.project_layout=go.kubebuilder.io/v3
 
+# should this operator be supported on OCP 4.4 and earlier (old appregistry format)
+LABEL com.redhat.delivery.backport: false
+# this is a bundle image and should be delivered via an index image
+LABEL com.redhat.delivery.operator.bundle: true
+
+
+LABEL name=aws-load-balancer-operator-bundle
+LABEL version="1.3.0"
+LABEL maintainer='NetworkEdge team <aos-network-edge-staff@redhat.com>'
+LABEL io.k8s.display-name='AWS Load Balancer Operator'
+LABEL io.k8s.description='Operator bundle for the AWS Load Balancer Operator'
+LABEL io.openshift.tags='networking,dns'
+LABEL com.redhat.component='aws-load-balancer-operator-bundle-container'
+LABEL com.redhat.license_terms='https://www.redhat.com/agreements'
+LABEL com.redhat.openshift.versions='v4.17'
+LABEL description='Operator bundle for the AWS Load Balancer Operator'
+LABEL distribution-scope='public'
+LABEL release='36'
+LABEL vendor='Red Hat, Inc.'
```

Comment:
`distribution-scope`, `release` and `vendor` labels are CPaaS specific (the first two are coming from the CPaaS build metadata). I don't see them defined in either [the midstream template](https://gitlab.cee.redhat.com/cpaas-midstream/aws-load-balancer-operator/-/blob/aws-lb-optr-1.2-rhel-8/distgit/containers/aws-load-balancer-operator-bundle/Dockerfile.in?ref_type=heads) or [bundle metadata](https://github.com/openshift/aws-load-balancer-operator/blob/main/bundle/metadata/annotations.yaml).